### ISSUE TITLE
Ошибка

### ISSUE THREAD

**Issue by icemuk** [friendly-traceback/friendly-traceback#234]
user_name = input("Привет, как тебя зовут? ")
user_age = input("Привет,", user_name,"а сколько тебе лет? ")
print("Создан новый пользователь. Имя: ", user_name, ", возраст: ", user_age)

**Comment by aroberge** [comment]
TypeError: input expected at most 1 argument, got 3

Case unknown; need to take care of it.

**Comment by icemuk** [comment]
Нашел решение:
user_name = input("Привет, как тебя зовут? ")
user_age = input("Привет,"+ user_name + "а сколько тебе лет? ")
print("Создан новый пользователь. Имя: ", user_name, ", возраст: ", user_age)

**Comment by aroberge** [comment]
Да, это решение вашей проблемы. Но объяснение необходимо добавить в Friendly-traceback, включая его перевод на разные языки.

---

Yes, it is a solution to your problem. But the explanation needs to be added to Friendly-traceback, including its translation in various languages.
